documentation/docs/instructor.md

```diff
@@ -1,82 +1,99 @@
-2. Click the environment definition you want to create VM definitions in.
-3. Select the `Create VM Definition` button towards the top right.
-4. Provide the name of the VM definition.
-5. Choose the type of image (*CentOS 7.3, Ubuntu Server 16.04.2, Windows Server 2012 R2*).
-6. Select the type of flavor (*01c-01m-10d reads 1 VCPU, 1 RAM GB, and 10 GB Total/Root Disk*).
-7. The timezone, hostname and Powershell script fields do not need to be complete in order to successfully create a VM definition. Please see *Cloud Configurations* to learn more about VM customizations.
-8. Press `Submit` when you have the necessary fields completed.
-
-## Update a VM Definition
-1. Go to the Environment Definitions List page by selecting the `Environment Definitions` tab found in the header.
-2. Choose the environment definition that has the VM definition you want to update.
-3. Find the VM definition and press the `Edit` button to modify any of the existing fields.
-
-## Delete a VM Definition
-1. Go to the Environment Definitions List page by selecting the `Environment Definitions` tab found in the header.
-2. Choose the environment definition that has the VM definition you want to delete.
-3. Find the VM definition and press the `Delete` button to delete it.
-
-## Create an Assignment
-1. From the home page, select the `Assignment` tab found in the header. You'll be taken to the Assignment List page which will contain all assignments created.
-2. While on this page, click the `Create Assignment` button near the top right corner.
-3. Provide a name, description, start and end date for the assignment, as well as choosing which course to add the assignment to along with the environment definition.
-4. After pressing `Submit`, you can spawn a fresh instance by selecting `Selecting a New Environment for this Assignment`.
-
-## Update an Assignment
-1. Go to the Assignment List page by clicking the `Assignments` tab found in the header.
-2. Find the assignment you want to update and select `Edit` in order to modify any of the existing fields.
-
-## Delete an Assignment
-1. Go to the Assignment List page by selecting the `Assignment` tab found in the header.
-2. Find the assignment you want to delete and click on `Delete`.
-
-## Timezone (Linux Only)
-Set the system timezone for an instance.
+## Becoming an Instructor User
+Contact your system administrator to have them promote you to become an instructor user.
+
+## Logging Into Cloud VCL
+Navigate to https://cloudvcl.cci.drexel.edu/ and log in using your Drexel credentials. You will know that you've successfully logged in when you see `Logged in as YourUsername` in the top right corner. Once you log in and arrive on the home page, if you see a blue text box that indicates you're an instructor, you've been successfully promoted. If not, please ask your system administrator to promote you. To log out, select the `Logout` button in the top right corner.
```

Comment:
Again, make this non-Drexel specific. Say instead, log in using your institution's credentials